### ISSUE TITLE
JS - step 6

### ISSUE THREAD

**Issue by saarangsoltani** [Samira64/Taskhero#7]
rename your function back to addTask, receiving a single task as its parameter.
update the function body to do the following
1) add a new `li` to `<ul class="task-list">`. The new 'li' does NOT need to have 3 'task-part' but it should have the task text as its content. So for example if you pass {id: 24, task: 'lets go to park', isComplete: false} to the addTask function, a new `<li> lets go to park</li>` should be added to the  `<ul class="task-list">`.

2) run the function with a sample `task` object. the task object should be in the format similar to the last exercise

in short this is how you code should look like

```javascript
function addTask(task)
{
  get the 'task-list' ul element and add a new li child to it. the li child should contain the task text
}

var myFirstTask = {id: 32, task: 'some sample task', isComplete: false}

addTask(myFirstTask)
```
http://www.w3schools.com/jsref/met_node_appendchild.asp
